product_packaging_type_pallet/models/product_packaging.py

```diff
@@ -3,15 +3,15 @@
 from odoo import fields, models
 
 
-class ProductPackagingType(models.Model):
-    _inherit = "product.packaging.type"
```

Comment:
Because of this, should we rename the module `product_packaging_type_pallet` to `stock_package_type_pallet`?
We also have already some `stock_package_type_*` modules (in v16) hosted in OCA/stock-logistics-warehouse. Should it be moved there?

@jbaudoux @rousseldenis what do you think?